parse/index.js

```diff
@@ -381,6 +386,49 @@ export default ({
     }
   }
 
+  function parseViewSetFlowToBasedOnData(block) {
+    let dataProp = block.properties.find((item) => item.name === 'data')
+    if (dataProp) {
+      let [context = null] = /\./.test(dataProp.value)
```

Comment:
can `context` be `null`? perhaps we should return if that's the case and print a warning?

Reply:
yes, I think should never be `null`. will add an error message in that case 👍